app/models/manageiq/providers/redhat/discovery.rb

```diff
@@ -1,19 +0,0 @@
-require 'manageiq/network_discovery/port'
-require 'ovirt'
-
-module ManageIQ
-  module Providers
-    module Redhat
-      class Discovery
-        def self.probe(ost)
-          ::Ovirt.logger = $rhevm_log if $rhevm_log
-
-          if ManageIQ::NetworkDiscovery::Port.open?(ost, ::Ovirt::Service::DEFAULT_PORT) &&
-             ::Ovirt::Service.ovirt?(:server => ost.ipaddr, :verify_ssl => false)
```

Comment:
Sorry this is my fault, I thought you were talking about a different "Probe" class - this one we need as far as I know, it is used to "discover" providers in a range of addresses.

Reply:
No the one we were discussing about was the probe of the OvirtSDK4. I thought that this one was not used. 
The problem with the discovery is the dependency from the "ovirt" gem. 
I think that I need to rewrite to use the OvirtSDK4.

Can I put back discovery.rb, the relative spec and the "ovirt" dependency?
I'll rewrite this part in another PR, removing also the "ovirt" gem. 
What do you think?

Reply:
sounds like the right thing to do, the only thing to note is that in that case this PR does not fully close the bug, only the next on will. Which is ok.
So please add this back and I will merge, and we will close the bug with the next PR.

Reply:
Yes, I agree. Basically the old gem was checking for SSL certificates at the predefined port 443. This checks can be obtained also with plain SSL requests without using the gem.

Reply:
@borod108 I solved the problem with Discovery with the OvirtSDK4. It's quite straightforward so if you agree I can add this also to the current PR without opening a new one.

Reply:
@gekorob awesome. Let me know when all the changes are in and we are ready for merge.

Reply:
@borod108 I thin we are ready. In the last commit I removed the ovirt gem and rewrote the code to use the OvirtSDK4 *exists?* that is part of the Probe class.